src/PMS/PMS5003T.cpp

```diff
@@ -110,33 +110,68 @@ bool PMS5003T::begin(void) {
 /**
  * @brief Read PM1.0 must call this function after @ref readData success
  *
- * @return int PM1.0 index
+ * @return int PM1.0 index (atmospheric environment)
  */
 int PMS5003T::getPm01Ae(void) { return pms.getPM0_1(); }
 
 /**
  * @brief Read PM2.5 must call this function after @ref readData success
  *
- * @return int PM2.5 index
+ * @return int PM2.5 index (atmospheric environment)
  */
 int PMS5003T::getPm25Ae(void) { return pms.getPM2_5(); }
 
 /**
  * @brief Read PM10.0 must call this function after @ref readData success
  *
- * @return int PM10.0 index
+ * @return int PM10.0 index (atmospheric environment)
  */
 int PMS5003T::getPm10Ae(void) { return pms.getPM10(); }
 
 /**
- * @brief Read PM 0.3 Count must call this function after @ref readData success
+ * @brief Read PM1.0 must call this function after @ref readData success
+ *
+ * @return int PM1.0 index (standard particle)
+ */
+int PMS5003T::getPm01Sp(void) { return pms.getRaw0_1(); }
+
+/**
+ * @brief Read PM2.5 must call this function after @ref readData success
+ *
+ * @return int PM2.5 index (standard particle)
+ */
+int PMS5003T::getPm25Sp(void) { return pms.getRaw2_5(); }
+
+/**
+ * @brief Read PM10 must call this function after @ref readData success
+ *
+ * @return int PM10 index (standard particle)
+ */
+int PMS5003T::getPm10Sp(void) { return pms.getRaw10(); }
+
+/**
+ * @brief Read particle 0.3 count must call this function after @ref readData success
  *
- * @return int PM 0.3 Count index
+ * @return int particle 0.3 count index
  */
 int PMS5003T::getPm03ParticleCount(void) {
   return pms.getCount0_3();
 }
 
+/**
+ * @brief Read particle 1.0 count must call this function after @ref readData success
```

Comment:
Do we really need to call it? Is it not more that we can if we want?

Reply:
Yes, totally agree. updated